composer.json

```diff
@@ -69,11 +69,13 @@
         "ci:dynamic": [
             "@ci:tests"
         ],
-        "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests",
-        "ci:php:stan": "phpstan --no-progress --configuration=config/phpstan.neon",
+        "ci:php:fixer": "\"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix --dry-run -v --show-progress=dots --diff bin src tests config",
+        "ci:php:lint": "find src tests config bin -name '*.php' -print0 | xargs -0 -n 1 -P 4 php -l",
```

Comment:
Please let's also have a line in the `scripts-description` section of the `composer.json`.

Reply:
Fixed.